build/Targets/Settings.targets

```diff
@@ -131,13 +141,21 @@
             Condition="Exists('$(RoslynDiagnosticsPropsFilePath)') And ('$(UseRoslynAnalyzers)' == 'true')" />
   </ImportGroup>
 
+  <!-- Otherwise, use the task from the bootstrap location -->
```

Comment:
Can we elaborate on why this is necessary?  I assume it's because the vanilla MSBuild version of these tasks can't handle the new properties.